pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,142 @@
+"""
+    Ophyd class for square-one tri-sphere motion system
+    (https://www.sqr-1.com/tsp.html)
+"""
+
```

Comment:
Your code doesn't work any more, needs to add
```
from __future__ import annotations
```
at the top here to allow the annotation | syntax in py39

Reply:
A small unit test would catch things like this, even one that simply imports the module